### DISCUSSION TITLE
Reducing MANPADS?

### DISCUSSION THREAD

**Discussion by Daek-wml** [Vdauphin/HeartsAndMinds#1615] [Question and answer] [answered]
Our pilots are facing an overwhelming number of MANPADS in the AO.As infantry we can hear 2-6 MANPADS being fired. Can we adjust the frequency of MANPADS? Thanks.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Sadly no, there is only an on and off option in mission parameter
You could check the content of `btc_type_units` to see how many AA type it has
Then you could remove some of them by using this https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L650-L671
Or instead add simple infantry type multiple time to reduce the probability to select a AA

Cheers

**Reply by Daek-wml** [reply]
I will try this method. 

Thanks
